python/cudf/cudf/_lib/column.pyx

```diff
@@ -251,8 +255,14 @@ cdef class Column:
 
     @property
     def children(self):
+        if (self.offset == 0) and (self.size == self.base_size):
+            return self.base_children
+        if self.base_children == ():
+            return ()
```

Comment:
Should we move these under the `if self._children is None` block and set `self._children` to these?

Reply:
fix'd